clouddriver-aws/src/test/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/loadbalancer/MigrateLoadBalancerStrategySpec.groovy

```diff
@@ -600,4 +601,60 @@ class MigrateLoadBalancerStrategySpec extends Specification {
     1 * amazonEC2.describeVpcs(_) >> new DescribeVpcsResult().withVpcs(new Vpc().withTags(new Tag("Name", "vpc1")))
     1 * sourceLookup.createSecurityGroup({ it.name == 'app-elb'}) >> new SecurityGroupUpdater(new SecurityGroup(groupId: 'sg-elb'), amazonEC2)
   }
+
+  @Unroll
+  void 'only adds Reference-Security-Policy attribute if present when creating policy'() {
```

Comment:
That's a good point, and it applies throughout the class/test suite. Moving forward, I will refactor these things (maybe even today) to only test what's under test.